rust/kcl-lib/tests/array_concat_non_array/ops.snap

```diff
@@ -127,61 +127,6 @@ description: Operations executed array_concat_non_array.kcl
     {
       "type": "GroupEnd"
     },
-    {
-      "type": "StdLibCall",
-      "name": "count",
```

Comment:
This call to std::array::count was previously included in the feature tree, but IMO it shouldn't have been.

Previously (on current ZDS staging):
<img width="799" height="637" alt="Screenshot 2025-10-17 at 11 06 58 PM" src="https://github.com/user-attachments/assets/a2a2fd32-61c1-4b10-a2a3-14d9af755ff7" />

Now (on this branch's ZDS build):

<img width="755" height="671" alt="Screenshot 2025-10-17 at 11 08 07 PM" src="https://github.com/user-attachments/assets/2b29fe0e-a4a2-4702-aaf5-1b7065d8b5ff" />